functions.php

```diff
@@ -387,7 +387,7 @@ public function enqueue_public_assets() {
 		// CSS files
 		wp_enqueue_style( 'bootstrap', $this->theme_dir . '/assets/lib/bootstrap/dist/css/bootstrap.min.css', array(), '4.0.0' );
 		wp_enqueue_style( 'slick', $this->theme_dir . '/assets/lib/slick-carousel/slick/slick.css', array(), '1.8.1' );
-		wp_enqueue_style( 'font-awesome', $this->theme_dir . '/assets/lib/font-awesome/css/font-awesome.min.css', array(), '4.7.0' );
+		wp_enqueue_style( 'font-awesome', $this->theme_dir . '/assets/lib/@fortawesome/fontawesome-free-webfonts/css/fontawesome.css', array(), '5.0.10' );
```

Comment:
is there a typo in @fortawesome? or is it correct?

Reply:
I had the same question, but it is an npm package
https://www.npmjs.com/package/@fortawesome/fontawesome-free-webfonts